stacktest/docker/docker.go

```diff
@@ -156,3 +188,7 @@ func updateEnv(setEnv map[string]string, env []string) []string {
 
 	return env
 }
+
+func IsDockerCompose() bool {
```

Comment:
This is a bit confusing since it doesn't really convey a difference between `docker-compose` and `docker compose`.  Maybe invert this and make it `IsNativeDocker` or something? I think this would also clean up usage as it seems like the result of this is always negated.

Reply:
Yeah good point, I was in a bit of a hurry. I'll clean it up